exasol/analytics/query_handler/query/select.py

```diff
@@ -135,6 +138,18 @@ def db_object_type(self) -> str:
     def db_operation_type(self) -> str:
         return self.audit_fields[DB_OPERATION_TYPE_TAG]
 
+    @property
+    def modifies_row_count(self) -> bool:
+        """
+        This property tells, whether the current ModifyQuery potentially
+        modifies the row count of a table. This is only relevant if the
+        ModifyQuery modifies a DbObjectType TABLE and is of DbOperationType
+        either INSERT or CREATE (with data of a subquery).
+        """
+        return (self.db_object_type == "TABLE") and (
+            self.db_operation_type in ["INSERT", "CREATE"]
+        )
+
```

Comment:
we also need a property which tells us, if the query is a CREATE OR REPLACE instead of a CREATE IF NOT EXISTS, because for a CREATE OR REPLACE TABLE, we need to log, also if the table already exists.